docs/content/docs/dev/table/sql/show.md

```diff
@@ -608,7 +608,7 @@ show tables;
 ## SHOW CREATE TABLE
 
 ```sql
-SHOW CREATE TABLE
+SHOW CREATE TABLE [catalog_name.][db_name.]table_name
```

Comment:
```suggestion
SHOW CREATE TABLE [[catalog_name.]db_name.]table_name
```

Reply:
Most doc is  `[catalog_name.][db_name.]table_name` . @RocMarshal

Reply:
https://github.com/apache/flink/blob/8c9b1767e08e3fdb3b60a0cc7f09a92647181d64/flink-table/flink-sql-client/src/test/resources/sql/catalog_database.q#L311